components/authz-service/storage/postgres/migration/sql/51_change_project_rules_query.up.sql

```diff
@@ -0,0 +1,54 @@
+BEGIN;
+
+CREATE OR REPLACE FUNCTION
+  query_rules_for_project(_project_id TEXT, _project_filter TEXT[])
+  RETURNS setof json AS $$
+
+  -- fetch staged rules
+   SELECT json_build_object(
+    'id', r.id,
+    'project_id', r.project_id,
+    'name', r.name,
+    'type', r.type,
+    'status', 'staged',
```

Comment:
do we need the `deleted` field here?

Reply:
or no bc these are only deleted . . . hmm ok

Reply:
it'd be more explicit, but then i'd also need to return any empty column from the applied table below. the API's going to ignore it so i think we can ignore it here